### ISSUE TITLE
Replacing single character variables with meaningful ones

### ISSUE THREAD

**Issue by onstash** [amoghpj/swar-synthesizer#9]
In order to make the code more readable, we could rename single character variables into something more meaningful.

**Comment by amoghpj** [comment]
Agreed. Will work on this now. Let me know if I forget some variables.

**Comment by onstash** [comment]
Let's label this issue as an enhancement.

**Comment by onstash** [comment]
Hi @amoghpj @reckoner165. You guys have more experience in music and musical notes than myself. So you'd know best what different single character variables mean and how they are being used. If we could transform all single character variables into something that's readable for people like me (who are not well versed with music and musical notes), I guess we could consider that a win. 😅
